index.rst

```diff
@@ -6,24 +6,499 @@
+A brief narrative introduction to the tutorial should be provided at the top of the page.
+
+The rest of the portal tutorial should be divided into sequentially numbered steps and substeps.
+Use descriptive text and screenshots to demonstrate what the user should do.
+Augment screenshots with indicators (e.g., arrows or circles) to guide the users attention as needed.
+
+Ensure that any Astronomical Data Query Language (ADQL) is put into code boxes in RST so that users may copy-paste whenever possible.
+
+It is very common, but not mandatory, to end all portal tutorials with a section called ``Exercises for the learner`` with suggestions of
+how the user can make changes to the tutorial test options and examples, or guide them on the next step forward on their own.
+
+
+Git branch, merge, and review policy for portal tutorials
+=========================================================
+
+The following applies when creating or updating tutorials in the `dp0-2_lsst_io repository <https://github.com/lsst/dp0-2_lsst_io>`_, which is part of the ``lsst`` GitHub Organization.
+
+Develop new tutorials, or update existing ones, in a new branch.
+This branch should be named for the corresponding Jira ticket (e.g., "tickets/PREOPS-12345").
+The new branch should be created from ``main``.
+Typically, only one tutorial is updated per ticket branch.
+
+Make commits and push changes to your branch in the ``dp0-2_lsst_io`` repository until work is complete, then open a pull request to ``main``.
+
+Contact one or more Rubin Observatory staff members with the appropriate expertise and ask them to review the notebook.
+At least one reviewer should be a member of the CET.
+If they agree, assign them as a reviewer on your pull request.
+
+Ensure that all of the reviewers' comments are addressed.
+Make changes and new commits to the branch, and respond to all of their comments with either a confirmation a change was made,
+or an explanation of why the request was not implemented.
+
+Contact the reviewers to let them know the pull request now awaits their approval.
+
+After the reviewers have approved the pull request, ``rebase and merge`` your ticket branch into the ``main`` branch.
+Resolve all conflicts, if there are any.
+After the successful merge, delete your branch.
+
+Remember to make comments in the associated Jira tickets about the major updates and mark the ticket as done.
+
+
+Stretch goals
+=============
+
+Work is on-going in these areas.
+
+Notebook metadata
+-----------------
+
+Embed notebook metadata (e.g., version, skills, packages) in a way that can be scraped and used to auto-generate the ``README.md`` file or a Table of Contents, to enable users to browse notebook contents.
+
+Narrative voice
+---------------
+
+Adopt a best practice for narrative instructions: should we use terms like "you," or should a passive voice be adopted?
```

Comment:
I vote against passive voice, unless there's a specific reason to use it.

Reply:
altered the sentence a little bit.  The narrative voice is something we are going to address in the future, so doesn't need to be defined here (that is my understanding at least)